### PR TITLE
Cors

### PR DESCRIPTION
**Pull request by TomWoodward**
enable cors, and fix swagger client issue

### CODE REVIEW

File: config/initializers/cors.rb [hunk 1]
```diff
@@ -5,12 +5,21 @@
...
-# Rails.application.config.middleware.insert_before 0, Rack::Cors do
-#   allow do
-#     origins 'example.com'
-#
-#     resource '*',
-#       headers: :any,
-#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
-#   end
-# end
+ALLOWED_CORS_ORIGINS = [
```
Comment (steveburkett) [inline config/initializers/cors.rb:8]: should prob. create controller tests for this (e.g., one test that allows openstax.org, and one that doesnt (a mock that raises a cors exception)

**Review by reedstrm — COMMENTED** [review]
Looks sane to me.

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -7,10 +7,13 @@ services:
       dockerfile: docker/Dockerfile
     ports:
       - "3001:3001"
-    command: bash -c "rm -f tmp/pids/server.pid && bin/rake setup && bin/rake about && bin/rails server -b '0.0.0.0' -p 3001"
+    command: bash -c "bin/rake about && bin/rails server -b '0.0.0.0' -p 3001"
```
Comment (jpslav) [inline docker/docker-compose.yml]: why remove the other commands?  the pid file can be left around (for no good reason) and will need to be removed before the server can start.  Guessing the setup task is to create the DB on first call.

File: config/initializers/cors.rb (hunk 1 above)
Comment (reedstrm) [reply config/initializers/cors.rb:8]: I think these got created Jan 5th, resolving,

File: config/secrets.yml.example [hunk 3]
```diff
@@ -11,11 +11,11 @@ development:
       encryption_private_key: 'foobar'
   kafka:
     application_name: 'events-capture-api'
-    broker_host: 'localhost:9092'
+    broker_host: <%= ENV['KAFKA_BROKER_HOST'] || 'localhost:9092' %>
```
Comment (reedstrm) [inline config/secrets.yml.example:14]: :+1:

**Review by reedstrm — APPROVED** [review]

File: docker/docker-compose.yml (hunk 2 above)
Comment (reedstrm) [reply docker/docker-compose.yml]: These where put back in the latest commit.

**Pull request merged**